### PR TITLE
Transaction timestamp

### PR DESCRIPTION
**Pull request by triump2020**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #3929

## What this PR does / why we need it:

### CODE REVIEW

**Review by jiangxinmeng1 — APPROVED** [review]

File: pkg/txn/clock/hlc.go [hunk 1]
```diff
@@ -136,6 +136,12 @@ func (c *HLCClock) Now() (timestamp.Timestamp, timestamp.Timestamp) {
 	return now, timestamp.Timestamp{PhysicalTime: now.PhysicalTime + int64(c.maxOffset)}
 }
 
+func (c *HLCClock) Get() timestamp.Timestamp {
```
Comment (lni) [inline pkg/txn/clock/hlc.go:139]: Is there any particular reason not to use HLC.Now()?

Comment (fengttt) [reply pkg/txn/clock/hlc.go:139]: The change itself looks fine -- but WHY do you ever need Get()?   How do you plan to use it?   I am not sure what is a valid use case.

Comment (fengttt) [reply pkg/txn/clock/hlc.go:139]: Wait, I spoke too soon.   I thought this was a small change about Get(), but not sure why it blown out to whole transaction timestamp.    I am not qualified to say the "change itself looks fine".

Comment (lni) [reply pkg/txn/clock/hlc.go:139]: When you want to use the same timestamp with the same value in different places, I think you should just pass that HLC timestamp around. When you want the current timestamp, just use Now(). 

Could you please provide a detailed use case on that Get() method? Thanks.

Comment (triump2020) [reply pkg/txn/clock/hlc.go]: Already removed the `clock.Get() ` method.

**Review by XuPeng-SH — APPROVED** [review]
lgtm

**Pull request merged**